homeassistant/components/cloudflare/config_flow.py

```diff
@@ -130,7 +130,7 @@ async def async_step_reauth_confirm(
             errors=errors,
         )
 
-    async def async_step_user(self, user_input: dict | None = None):
+    async def async_step_user(self, user_input: dict | None = None) -> FlowResult:
```

Comment:
I'm sure that the type hint for user_input could also be improved, but there are so many possibilities for `async_step_***` that I'm not sure it would be useful to implement via the pylint plugin.
I think just enforcing the FlowResult is already an improvement...